modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+
+. xref:learn:clusters-and-availability/xdcr-viewing-conflict-logs.adoc[Viewing Conflict Logs]: When XDCR detects conflicts, it logs xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record] into the conflict collection specified by you.
+You can access and review the logged conflicts.
+
+[#conflict-collection]
+== Conflict Collection
+
+A conflict collection is a regular collection that you create to store the xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record] (Conflict Record Document, and a copy of the source and target documents responsible for the conflict) from the affected clusters of the XDCR topology.
+
+For information about configuring a collection to log conflicts, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#configure-conflictlogging-settings[Configuring the conflictLogging Settings].
+
+* As conflict collection is a regular collection, you can use regular RBAC to control standard access and programmatic access to the documents in the collection.
+
+* You can store conflict collection in the source bucket or in a separate collection in a bucket that's not used in any XDCR replications.
+
+* XDCR can write into conflict collections associated with the source cluster, however, XDCR cannot modify the conflict collections in any target clusters.
+
+NOTE: XDCR adds a hidden system xattribute flag called `_xdcr_conflict` into each of the three documents in a xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record] stored in conflict collection, which prevents these documents (conflict logs) from replicating to the target.
+
+[#conflict-record]
+=== Conflict Record
+
+A conflict record consists of the following three documents:
+
+* *Conflict Record Document (CRD):* It's a JSON record document that contains source and target document key-value pairs.
+CRD consists of the following elements:
+** Metadata document.
+** Document key: `crd_<timestamp>_<SHA>`.
+** Source document key and target document key. 
+
+* *Source document:* It's a copy of the source document that caused the conflict.
+This logged source document consists of the following elements:
+** Document body and xattrs of the source bucket from the conflict.
+** Document key: `src_<timestamp>_<SHA>`.
+* *Target document:* It's a copy of the target document that caused the conflict.
+This logged target document consists of the following elements:
+** Document body and xattrs of the target bucket from the conflict.
+** Document key: `tgt_<timestamp>_<SHA>`.
+
+The generated `<SHA>` ensures that each conflict for the same document is unique and logs each conflict for each document key.
```

Comment:
The generated `<SHA>` ensures that each conflict event for the same document (same document id) is logged uniquely.